zeebe/broker/src/main/java/io/camunda/zeebe/broker/exporter/stream/ExporterMetrics.java

```diff
@@ -18,6 +19,23 @@ public final class ExporterMetrics {
   private static final String LABEL_NAME_ACTION = "action";
   private static final String LABEL_NAME_VALUE_TYPE = "valueType";
   private static final String NAMESPACE_ZEEBE = "zeebe";
+
+  private static final Histogram EXPORTING_LATENCY =
+      Histogram.build()
+          .namespace(NAMESPACE_ZEEBE)
+          .name("exporting_latency")
+          .help("Time between a record is written until it is picked up for exporting (in seconds)")
+          .labelNames(LABEL_NAME_PARTITION, LABEL_NAME_VALUE_TYPE)
+          .register();
+
+  private static final Histogram EXPORTER_LATENCY =
```

Comment:
:wrench:  The term `latency` is more suitable to represent queuing time. So here it is better to use a different term, preferably `duration`. So may be something like `RECORD_EXPORTING_DURATION`.

Reply:
Yes I was also unsure during writing tbh. Found a good/simple enough definition: `latency is how long it takes to start the behavior and duration is how long the behavior lasts.`